app/routes/traffic-measure-concepts/index.js

```diff
@@ -25,6 +25,33 @@ export default class TrafficMeasureConceptsIndexRoute extends Route {
       query['filter[label]'] = params.code;
     }
 
-    return await this.store.query('traffic-measure-concept', query);
+    const result = await this.store.query('traffic-measure-concept', query);
```

Comment:
you may want to consider including the template here, avoids having to request it later on